src/FunctionApps/DSTP/src/main/kotlin/gov/cdc/prime/phdi/Utilities/HL7Reader.kt

```
@@ -0,0 +1,36 @@
+package gov.cdc.prime.phdi.Utilities
+
+import org.apache.logging.log4j.kotlin.Logging
+import java.time.format.DateTimeFormatter
+import java.io.InputStream
+import java.io.OutputStream
+import java.nio.charset.StandardCharsets
+
+public class HL7Reader() {
+    fun readHL7MessagesFromFile(
+        filepath: String
+    ): String {
+        // reads in a file of one or more HL7 messages and returns the content as a string
+        return "This method is not currently implemented."
```

this should raise an error, not return a dummy string, no? ditto below.